server/services/utils/addressHelpers.js

```diff
@@ -1,52 +1,19 @@
-const {getIn, replaceArrayItem, mergeWithRight, removeFromArray} = require('../../utils/functionalHelpers');
+const recordList = require('./recordList');
```

Comment:
I'd recommend doing away with addressHelpers altogether seeing as it is practically empty. Would just put these functions in the service accessing the recordList util